src/modules/cp/const/index.ts

```diff
@@ -2,22 +2,22 @@ import { i18n } from 'next-i18next';
 
 export const CP_ADMIN_NAVIGATION_LINKS = (): {
 	id: string;
-	label: string | undefined;
+	label: string;
 	href: string;
 }[] => [
 	{
 		id: 'requests',
-		label: i18n?.t('modules/cp/const/index___aanvragen'),
+		label: i18n?.t('modules/cp/const/index___aanvragen') ?? '',
 		href: '/beheer/aanvragen',
 	},
 	{
 		id: 'visitors',
-		label: i18n?.t('modules/cp/const/index___bezoekers'),
+		label: i18n?.t('modules/cp/const/index___bezoekers') ?? '',
 		href: '/beheer/bezoekers',
 	},
 	{
 		id: 'settings',
-		label: i18n?.t('modules/cp/const/index___instellingen'),
+		label: i18n?.t('modules/cp/const/index___instellingen') ?? '',
```

Comment:
don't these automatically output the key if the translation value isn't found? So the `?? ''` case will never be used.

In avo we used i18next (not related to nextjs) to do the translations and that had an option to set values for keys that were not found:
```
		parseMissingKeyHandler: (key) => {
			if (key.includes('___')) {
				return `${upperFirst(lowerCase(key.split('___').pop()))} ***`;
			}
			return `${key} ***`;
		},
```

maybe nextjs has something similar?

Reply:
The problem is with the i18n instance from next-i18next, the type can be undefined and the `label` prop is expecting a string that's why I added the `?? ''` but I agree that it isn't the best solution. Didn't have the time to find a better solution.